vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -55,7 +55,7 @@ namespace Microsoft.ReactNative
 
   [webhosthidden]
   DOC_STRING("Enables a view manager to create views whose behavior depend on the the property values passed to the view manager at creation time. \
-  For example, a view manager could choose to create different types of UI elements based on the properties passed in.")
+For example, a view manager could choose to create different types of UI elements based on the properties passed in.")
```

Comment:
intentional? Other multiline have used the `+` and "\n" pattern..

Reply:
yes intentional